public-api/src/main/java/org/pmiops/workbench/cdr/model/DbMeasurementConceptInfo.java

```diff
@@ -20,17 +20,17 @@
 
 @Entity
 @Table(name = "measurement_concept_info")
-public class MeasurementConceptInfo {
+public class DbMeasurementConceptInfo {
     private Long conceptId;
     private int hasValues;
 
     @OneToOne(fetch=FetchType.LAZY)
     @JoinColumn(name="concept_id")
-    private Concept concept;
+    private DbConcept dbConcept;
```

Comment:
nit: please remove since it's never used

Reply:
This join relationship is used here https://github.com/all-of-us/data-browser/blob/master/public-ui/src/app/data-browser/views/ehr-view/ehr-view.component.ts#L311 to filter lab values vs orders.

Reply:
`dbConcept` is never referenced in this class and can be removed. `DbConcept` will never be referenced from the UI.

Reply:
Removed this.